application/F3DStarter.cxx

```diff
@@ -85,6 +85,7 @@ class F3DStarter::F3DInternals
     double MaxSize;
     std::optional<double> AnimationTime;
     bool Watch;
+    int FrameRate;
```

Comment:
Let's make it a double.  
Framerates like 29.97fps can be found in DCC tools like Maya (for legacy reasons, but still).

Reply:
Another example with my monitor settings on Windows:
![image](https://github.com/user-attachments/assets/83a8fe7b-d9d9-4b51-b6bc-697a2ef1c927)